ee/api/test/test_organization.py

```diff
@@ -39,25 +39,25 @@ def test_create_two_similarly_named_organizations(self, mock_choice):
             {"name": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"},
         )
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertDictContainsSubset(
+        self.assertLessEqual(
```

Comment:
we really should switch to the `assert foo == bar` syntax everywhere...  (we get much nicer output on fail)

at some point i'll add a linter and do one big change